core/server/api_container/server/api_container_service.go

```diff
@@ -671,6 +672,11 @@ func (apicService ApiContainerService) runStarlarkPackageSetup(packageId string,
 
 func (apicService ApiContainerService) runStarlark(parallelism int, dryRun bool, packageId string, serializedStarlark string, serializedParams string, stream grpc.ServerStream) {
 	responseLineStream := apicService.startosisRunner.Run(stream.Context(), dryRun, parallelism, packageId, serializedStarlark, serializedParams)
+	startosis_warning.Setup(stream)
```

Comment:
I tried testing this approach multiple times, including running starlark script multiple times in same enclave and didn't see any issues. Let me know if you know of any other tests I should run @gbouv 

StartosisRunner.Run(...) uses chan to co-ordinate between various go-routines. I tried to move this in that function but it seems a complex and to be honest not sure what value we would be gaining. 

I think I would need to add a new warning chan and forward the message from that channel to `starlarkResponseLines` channel, and this creates unnecessary dependency on the later channel as it is required for collecting the output from each go-routines, whereas warning streams to me is more of a global module which can be called anytime inside a go-routine.